qiskit/tools/parallel.py

```diff
@@ -102,6 +102,8 @@ def parallel_map(  # pylint: disable=dangerous-default-value
         terra.parallel.update: One of the parallel task has finished.
         terra.parallel.finish: All the parallel tasks have finished.
     """
+    if not values:
```

Comment:
Looks like this is causing a failure in the docs build. From https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=15769&view=logs&j=23e88340-9b01-571b-4128-239215012d30&t=fd5b7ae3-9f70-5b67-b24e-b010b8098eb2&l=88 :

```
Sphinx parallel build error:
sphinx.errors.ExtensionError: Cell raised uncaught exception:
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1-42cd38aec8f4> in <module>
      4 
      5 get_ipython().run_line_magic('qiskit_progress_bar', '')
----> 6 parallel_map(np.sin, np.linspace(0,10,100));

~/work/1/s/qiskit/tools/parallel.py in parallel_map(task, values, task_args, task_kwargs, num_processes)
    103         terra.parallel.finish: All the parallel tasks have finished.
    104     """
--> 105     if not values:
    106         return []
    107     if len(values) == 1:

ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
##[error]The operation was canceled.
```

Reply:
Oh I didn't realize we were passing in value lists as numpy arrays too. I'll change this to `if len(values) == 0` then

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4633/commits/7fc52b76726c792a9d5c03bfa377bdf34f6f0f34